apps/prairielearn/python/prairielearn.py

```diff
@@ -1692,7 +1726,7 @@ def clean_identifier_name(name):
     return name
 
 
-def load_extension(data, extension_name):
+def load_extension(data: QuestionData | Any, extension_name: str) -> Any:
```

Comment:
Does this work without the Any in the union? An Any in a type union can always be simplified to just an Any.

Reply:
This means that

```
    if "extensions" not in data:
        raise ValueError("load_extension() must be called from an element!")
```

This block would never run. I wanted to keep the `Any` to highlight the intended type.

> Type analysis indicates code is unreachable, Pylance

I am not sure what best practice is here, if you still feel it should be removed I can.

Reply:
Ohhh I see. I think it's ok to keep error messages like that with an ignore comment. In general, we want annotations to reflect intended behavior (i.e. you're not supposed to use an int with this function).

Reply:
Fixed in 487683782.